icons/apisec.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We'll need a (roughly) square logo for it to work in the UI for the setup page. I think just the shield portion of this might work well...? 🙂